cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -0,0 +1,78 @@
+{# ==========================================================================
+
+   related_posts.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates related posts markup when given:
+
+   settings:       An object used to customize the markup.
+
+   settings.posts: An array of related posts.
+
+   ========================================================================== #}
+
+{% macro render(settings) %}
+{% from 'macros/util/format/url.html' import slugify as slugify %}
+{% set defaults = {
+    'search_types': 'posts, newsroom, press release speech, testimony',
```

Comment:
This looks like it's missing a comma? `'posts, newsroom, press release, speech, testimony',`